packages/hooks/src/useGroupedTransactions.ts

```diff
@@ -2,7 +2,7 @@ import _ from 'lodash';
 import moment from 'moment';
 import { Statement } from '@deriv/api-types';
 
-const useGroupedFiatTransactions = (transactions: Statement['transactions']) => {
```

Comment:
@rostislav-deriv This hook doesn't have any state, I think you can make it a utility function inside `@deriv/utils` and remove this hook 🙇🏻

Reply:
Moved the logic to `utils`